Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,43 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Start program
+            Console.WriteLine("Hello World! Welcome to the first C# 2020 Bootcamp Homework.");
+            Program.HW1();
+            Console.WriteLine("Okay, Person #2:");
+            Program.HW1();
+        }
+        static double GetBMI(double weight, int height) //calculates the BMI by receiving weight and height as parameters and dividing weight by height squared (kg/m^2)
+        {
+            //Function Declarations
+            double BMI, hSquared;
+
+            //convert cm to meters and square
+            hSquared = (height / 100d);
+            hSquared *= hSquared;
+
+            BMI = weight / hSquared;
+
+            return BMI;
+        }
+        static void HW1()
+        {
+            //Variable Declarations
+            string name, surname;
+            int age, height; //height in cm
+            double weight; //weight in kg
+
+            Console.WriteLine("What is your first name?");
+            name = Console.ReadLine();
+            Console.Write("Hello, " + name + '!' + " I just need a little extra information from you.\nSurname: ");
+            surname = Console.ReadLine();
+            Console.Write("Age: ");
+            age = Convert.ToInt32(Console.ReadLine()); //Console.ReadLine returns a string, so you must convert it to the appropriate type.
+            Console.Write("Weight(in kg): ");
+            weight = Convert.ToDouble(Console.ReadLine());
+            Console.Write("Height(in cm): ");
+            height = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("\n" + name + ' ' + surname + " is " + age + " years old. Their weight is " + weight + "kg, and their height is " + height + "cm tall. This would make their BMI " + GetBMI(weight, height) + '.'); //bug @ Convert.ToString
```

Comment:
Please note that you're appending a character to the string in your Console.WriteLine (line 45). The character is a space. It is better to make this a string, so you cannot make a mistake such as:
`
Console.WriteLine("The answer is: ");`
`Console.WriteLine(3 '&' + 7);`
Please run this code yourself. What do you expect it to print? Obviously, you want it to output: 

> The answer is: 
> 3&7

However, because your adding an integer with the character in the second Console.WriteLine, you will end up with a numerical value where & is being represented by its ASCII value, as explained in the first stream. As a reference, have a look at http://www.asciitable.com/. You will find its value of 38, and thus the answer will be 48. That's not what you wanted!
This is a long review about just a character instead of a string, but I'm trying to explain why you should be careful because it can easily lead to a bug. Besides that, it's also strange to use strings everywhere and then suddenly a character.